roles/openshift_storage_glusterfs/defaults/main.yml

```diff
@@ -5,22 +5,22 @@
 # becomes "oreg.example.com/rhgs3/rhgs-server-rhel7:latest"
 l_gluster_image_dict:
   origin: "docker.io/gluster/gluster-centos:latest"
-  openshift-enterprise: "registry.access.redhat.com/rhgs3/rhgs-server-rhel7:latest"
+  openshift-enterprise: "registry.redhat.io/rhgs3/rhgs-server-rhel7:latest"
```

Comment:
@sdodson this is another case that needs to be examined... will these values get replaced from oreg_url?

Reply:
These are defaults.  They will be affected if someone sets oreg_url.

Reply:
cool